docker-compose.yml

```diff
@@ -92,6 +93,8 @@ services:
       RABBITMQ_USER: ${RABBITMQ_USER}
       RABBITMQ_PASS: ${RABBITMQ_PASS}
       DOCUMENT_MANAGER_DB_PASSWORD: ${DOCUMENT_MANAGER_DB_PASSWORD}
+    depends_on:
+      - document-manager-db
```

Comment:
This has nothing to do with the user story right?
It has just been forgotten when we introduced the document manager service, correct?

Reply:
What just came to my mind: should we add a dependency on rabbit-mq for all spring services? Sometimes when you start a spring service and it starts quicker than rabbit-mq it throws an exception that it could not establish a connection to rabbit-mq. However, it retries until it works.

Reply:
You can do it but I don't think that it will help a lot:
With `depends_on` Docker Compose ensures that the service with dependencies starts before the dependent service starts, ok. But it does not wait for the dependent service to be fully "healthy" or ready. So the exceptions will probably occur anyways.

What do you think?

Reply:
Decided with Florian not to add "depends_on" for rabbit mq service